akka-actor-typed-tests/src/test/scala/akka/actor/typed/scaladsl/adapter/AdapterSpec.scala

```diff
@@ -157,6 +157,36 @@ class AdapterSpec extends AkkaSpec {
 
       typed1 should be theSameInstanceAs typed2
     }
+
+    "not crash if guardian is stopped" in {
+      for { _ ← 0 to 10 } {
+        var system: akka.actor.typed.ActorSystem[NotUsed] = null
+        try {
+          system = ActorSystem.create(Behavior.stopped[NotUsed], "AdapterSpec-stopping-guardian")
```

Comment:
Should we even allow stopped behaviors as guardians? I'd suggest not to allow these.

Reply:
But even then we might still have a stopped behavior below deferred, but even that might be easier to handle than a directly stopped one that directly shuts down the ActorAdapter right after having started.